.vscode/settings.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Can we revert the inclusion of this file?

Reply:
done